content/dstu2/medication-statement.md

```diff
@@ -26,3 +30,106 @@ Search for MedicationStatements that meet supplied query parameters:
 
 <%= headers 200 %>
 <%= json(:dstu2_medication_statement_bundle) %>
+
+## Create
+
+Create new MedicationStatement. The medicationReference field must refer to a contained Medication which much have the code field populated.
```

Comment:
"which much have" => "which must have"

However, I think the first must is sufficient: The medicationReference field must refer to a contained Medication with the code field populated.